test/jdk/ProblemList.txt

```diff
@@ -818,7 +818,10 @@ sun/tools/jstat/jstatLineCounts2.sh                             8268211 linux-aa
 sun/tools/jstat/jstatLineCounts3.sh                             8268211 linux-aarch64
 sun/tools/jstat/jstatLineCounts4.sh                             8268211 linux-aarch64
 
-sun/tools/jmap/BasicJMapTest.java                               8274245 generic-all
+sun/tools/jmap/BasicJMapTest.java#G1                            8274245 generic-all
+sun/tools/jmap/BasicJMapTest.java#Parallel                      8274245 generic-all
+sun/tools/jmap/BasicJMapTest.java#Serial                        8274245 generic-all
+sun/tools/jmap/BasicJMapTest.java#Z                             8274245 generic-all
```

Comment:
I didn't think it was necessary to enumerate subtests like this?

Reply:
I didn't think so either, but the original ProblemListing bug:

JDK-8274294 ProblemList sun/tools/jmap/BasicJMapTest.java

didn't do the job.